src/ray/gcs/gcs_server/gcs_kv_manager.h

```diff
@@ -32,38 +32,46 @@ class InternalKVInterface {
  public:
   /// Get the value associated with `key`.
   ///
+  /// \param ns The namespace of the key.
   /// \param key The key to fetch.
   /// \param callback Callback function.
-  virtual void Get(const std::string &key,
+  virtual void Get(const std::string &ns, const std::string &key,
                    std::function<void(std::optional<std::string>)> callback) = 0;
 
   /// Associate a key with the specified value.
   ///
+  /// \param ns The namespace of the key.
   /// \param key The key for the pair.
   /// \param value The value for the pair.
   /// \param overwrite Whether to overwrite existing values. Otherwise, the update
   ///   will be ignored.
   /// \param callback Callback function.
-  virtual void Put(const std::string &key, const std::string &value, bool overwrite,
+  virtual void Put(const std::string &ns, const std::string &key,
```

Comment:
@ericl  I don't want to use `namespace_` here. Maybe it's ok to use ns here. Any suggestions?